### PR TITLE
[tests-only][full-ci] Removed closed sharing issues form various features

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
With this PR:
- closed sharing issues https://github.com/owncloud/product/issues/207 & https://github.com/owncloud/ocis/issues/1303 are removed from various feature files
- scenarios updated with open and more pointed issues

## Motivation and Context
do not use closed issues tags for test scenarios

## How Has This Been Tested?

- 👨🏼‍🚀 
- 🤖 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17075/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

File: tests/acceptance/helpers/sharingHelper.js [hunk 1]
```diff
@@ -130,6 +130,7 @@ module.exports = {
             } else if (expectedDetail.field === 'expiration') {
               expectedDetail.value = sharingHelper.calculateDate(expectedDetail.value)
             }
+            console.log(expectedDetail)
```
Comment (SwikritiT) [inline tests/acceptance/helpers/sharingHelper.js:133]: Is this needed?

File: tests/acceptance/stepDefinitions/sharingContext.js [hunk 2]
```diff
@@ -180,10 +180,12 @@ const shareFileFolder = async function(
     }
   }
   const url = 'apps/files_sharing/api/v1/shares'
+  console.log(params)
```
Comment (SwikritiT) [inline tests/acceptance/stepDefinitions/sharingContext.js:183]: This and the one below too.

File: tests/acceptance/helpers/sharingHelper.js (hunk 1 above)
Comment (kiranparajuli589) [reply tests/acceptance/helpers/sharingHelper.js]: not needed. removed!

File: tests/acceptance/stepDefinitions/sharingContext.js (hunk 2 above)
Comment (kiranparajuli589) [reply tests/acceptance/stepDefinitions/sharingContext.js]: removed

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17091/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17092/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Review by phil-davis — APPROVED** [review]

**Pull request merged**